### PR TITLE
Fix isomorphic rendering for JSS 7.

### PR DESCRIPTION
**Pull request by phyllisstein**
This PR bumps the built-in JSS dependency to `7.1.1` and updates the `reset` function to use the new version's API. This unbreaks server-side rendering for the most part, though the `<style>` tags being rendered into the DOM are still empty both on the server and in the client---more investigation is necessary to figure out why.

### CODE REVIEW

**Comment by kof** [comment]
If they seem empty on the client only it might be the current state of the browser implementation. They don't show rules inserted using insertRule api in the dom. I fits on the server as well, something is broken here.

**Comment by kof** [comment]
I would prefer to find out the reason first and then merge.

**Comment by phyllisstein** [comment]
That seems perfectly sensible! They're empty on the server as well, so I'll see if I can figure out why.

**Comment by phyllisstein** [comment]
Change just pushed fixes null `<style>` tags on the client. They were empty on the server because I was importing a different instance of `aphrodite-jss` 🤕 🌴 .

File: src/index.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ export default function aphrodisiac(jss, options) {
 
     const style = rules.reduce(mergeStyles, {})
     sheet.addRule(className, style, {className})
+    sheet = sheet.deploy()
```
Comment (kof) [inline src/index.js:31]: why do you need deploy? You may be trapped by the fact that insertRule api doesn't update the devtools representation, so you don't see the rules, but they are there, to see them you can go to the `style.sheet.rules`

File: src/index.js [hunk 2]
```diff
@@ -44,7 +45,7 @@ export default function aphrodisiac(jss, options) {
 
   function reset() {
     sheet.detach()
-    jss.sheets.remove(sheet)
+    jss.removeStyleSheet(sheet)
```
Comment (kof) [inline src/index.js:48]: removeStyleSheet already does sheet.detach() see https://github.com/cssinjs/jss/blob/master/src/Jss.js#L61

**Comment by kof** [comment]
Just merged another PR with the update to jss 7, did it fix the issue?

**Comment by kof** [comment]
Seems to be fixed, please reopen if not.
